images.yaml

```diff
@@ -374,6 +374,24 @@
     - tagSuffix: giantswarm
       dockerfileOptions:
       - RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
+- name: k8s.gcr.io/kube-apiserver
```

Comment:
This image is only used to be notified when there's a new version and to extract the version by running `kube-apiserver --version` in the `Dockerfile`. It's not pretty, but it works.